ddpui/settings.py

```diff
@@ -33,24 +35,36 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = os.getenv("DEBUG", "") == "True"
 
+
 # CORS
+
 ALLOWED_HOSTS = [
     "localhost",
     "127.0.0.1",
     "staging-api.dalgo.in",
     "api.dalgo.in",
+    "host.docker.internal",
+    "backend-image-fosambcxuq-uc.a.run.app",
```

Comment:
what are these two hosts?

Reply:
This is our hosts which I will remove. I think in the next PRs we will move this to .env since it will be different for each deployment